tests/glade/run_glade_tests.sh

```diff
@@ -5,6 +5,18 @@ if ! type parallel 2>&1 > /dev/null; then
     exit 99
 fi
 
+if ! rpm -q gnome-icon-theme 2>&1 > /dev/null; then
```

Comment:
I think ``! rpm -q gnome-icon-theme &> /dev/null;`` would be better here and few lines below too.